scripts/update_gene_symbols.py

```diff
@@ -109,27 +115,32 @@ def hgnc_tsv_to_dict(hgnc_file, old_sym, new_sym):
                     sym_dict[old_symbol] = [new_sym_info]
     return sym_dict
 
-def update_fusions_tsv(fusions_file, sym_dict, update_columns, out_file, retain_records, explode_records):
-    """Reads lines of the fusions_file, updates the genes, and writes to an output
+def update_input_tsv(input_tsv, sym_dict, update_columns, out_file, retain_records, explode_records, fake_columns):
+    """Reads lines of the input_tsv, updates the genes, and writes to an output
 
     Iterates through the custom fusions file and feeds each line to the line processor.
     Takes the output of the line processor and writes that to the output file.
 
     Args:
-        fusions_file: An unopened TXT or TXT.GZ file that contains Custom Fusion information
+        input_tsv: An unopened TXT or TXT.GZ file that contains Custom Fusion information
         sym_dict: A dict with old symbol keys and corresponding value list of new symbols
-        update_columns: A list of columnnames in the fusions_file that must be updated
+        update_columns: A list of columnnames in the input_tsv that must be updated
         out_file: A string filename (files ending in GZ will be compressed) for the output
         retain_records: A boolean that indicates if original records should be kept
         explode_records: A boolean that enables multirecord outputs
 
     Returns:
         None
     """
-    with (gzip.open if fusions_file.endswith("gz") else open)(fusions_file, "rt", encoding="utf-8") as f:
+    # output to STDERR old -> new gene changes
+    print ('{}\t{}'.format("old gene", "new gene"), file=sys.stderr)
+    with (gzip.open if input_tsv.endswith("gz") else open)(input_tsv, "rt", encoding="utf-8") as f:
         with (gzip.open if out_file.endswith("gz") else open)(out_file, "wt", encoding="utf-8") as w:
-            header = f.readline().strip().split('\t')
-            w.write('\t'.join(header) + '\n')
+            if fake_columns is None:
+                header = f.readline().strip().split('\t')
+                w.write('\t'.join(header) + '\n')
+            else:
+                header = '\t'.join(fake_columns)
```

Comment:
We want to print the provided header to the output, no?
```suggestion
            if fake_columns is None:
                header = f.readline().strip().split('\t')
            else:
                header = '\t'.join(fake_columns)
            w.write('\t'.join(header) + '\n')
```

Reply:
No, I don't actually want that. If the input did not have headers to begin with, then the output should remain as such. This is simply to allow the downstream functions to know which columns to act on